src/vellum/workflows/graph/graph.py

```diff
@@ -38,6 +40,7 @@ def __rshift__(self, other: "GraphTarget") -> "Graph":
     "Port",
     "Graph",
     "NoPortsNode",
+    "BaseTrigger",
```

Comment:
[blocking] I wouldn't expect BaseTrigger to be a valid `GraphTarget`, since we don't want to allow `Node >> Trigger`